fastlane/Fastfile

```diff
@@ -130,7 +130,8 @@ def build_ios_e2e
         destination: "generic/platform=iOS Simulator",
         # 2. fixing compilations issues as stated in https://stackoverflow.com/a/20505258
         #    it looks like i386 isn't supported by React Native
-        xcargs: "ARCHS=\"x86_64\" ONLY_ACTIVE_ARCH=NO",
+        # -UseModernBuildSystem=N is a temporary fix for Xcode 10.1
+        xcargs: "ARCHS=\"x86_64\" ONLY_ACTIVE_ARCH=NO -UseModernBuildSystem=N",
```

Comment:
Actually, we could move `UseModernBuildSystem=N` to `export_options` like here:
https://github.com/status-im/status-react/blob/22bd2f1761deba99e7c5b2d47b4345474bcde1e4/fastlane/Fastfile#L216-L222

Reply:
That section of the file actually has it in both places, should I do the same?

Reply:
Yes. It's redundant. And `xcargs` is more of a hack.